tests/end2end/data/fileselect.html

```diff
@@ -17,5 +17,6 @@
     <body onload="onLoad()">
         <input type="file" id="single_file" name="single_file" accept=".txt">
         <input type="file" id="multiple_files" name="multiple_files" accept=".txt" multiple>
+        <input type="file" id="folder" name="folder" webkitdirectory>
```

Comment:
Maybe add some text here (above the inputs) as well, so that the page is a bit easier to use for manual testing as well? Right now it's a bit confusing:

![image](https://user-images.githubusercontent.com/625793/110362855-1d239480-8042-11eb-8ca6-6227e3c8e8f1.png)